src/app/pages/account/account-navigation/account-navigation.items.b2c.ts

```diff
@@ -0,0 +1,91 @@
+import { NavigationItem } from './account-navigation.component';
+
+// not-dead-code
+export const navigationItems: NavigationItem[] = [
+  { id: 'my-account', localizationKey: 'account.my_account.link', routerLink: '/account' },
+  {
```

Comment:
even if you switch on b2b features in the environment.properties they won't work in b2c - so imo it is not necessary to leave them in this file: requisitions, approvals, costcenters, user management